python/felis/datamodel.py

```diff
@@ -130,7 +133,7 @@ class Column(BaseObject):
     length: int | None = None
     """The length of the column."""
 
-    nullable: bool = True
+    nullable: bool | None = None
```

Comment:
Maybe explain what `None` means in this context?

Reply:
> Maybe explain what `None` means in this context?

The existing logic is that if `nullable` is not provided explicitly, it is set to `False` for columns with a numeric type. Otherwise, it is set to `True`. The `None` value is used to signify that no value was provided in the schema itself so this default logic is triggered. So it probably needs to be left alone for now unless we want to change the existing default behavior.

Reply:
What I would like is that this is added to the docstring.

Reply:
> What I would like is that this is added to the docstring.

I've added a docstring to the field for clarification on handling of the default `None` value.

Reply:
I will leave this for now. There has been some discussion about changing this default to `False`, but I do not want to do that on this ticket.